src/span-data.ts

```diff
@@ -120,6 +121,16 @@ export class RootSpanData extends BaseSpanData {
     this.span.kind = SpanKind.RPC_SERVER;
   }
 
+  createChildSpan(options: SpanOptions): SpanData {
+    options = options || {name: ''};
```

Comment:
Is `options` optional? Why do you need this line?

Reply:
They're optional, but in the absence of type checking it prevents an error from being thrown when options aren't passed in. It's sort of a relic of older versions of this module -- I added it to keep it consistent with the same precondition check in `traceApi.createChildSpan`.

Reply:
Erm... I'm not familiar with the history of this module, but why not mark it as optional then?

Reply:
The "history of this module" is just that it was once written in JavaScript :)

I've marked `options` as optional.